packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1289,7 +1290,10 @@ export function purposeServiceBuilder(
         consumerDelegation
       );
 
-      if (purposeIsDraft(purposeToClone.data)) {
+      if (
+        purposeIsDraft(purposeToClone.data) ||
+        purposeIsArchived(purposeToClone.data)
```

Comment:
```suggestion
        isClonable(purposeToClone.data)
```
Instead of checking for specific versions I would create a new validator like this